test/fixtures/simple_example/outputs.tf

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+# These outputs are fed to inspec as inputs.
+output "project_id" {
```

Comment:
Since these are new outputs let's follow the pattern of documenting all variables and outputs in all modules.

Reply:
Good catch, fixed in 60fb977...5cdbb87